ui/src/app/pages/access/data-access-requirements.spec.tsx

```diff
@@ -58,11 +58,21 @@ const oneYearAgo = () => nowPlusDays(-EXPIRY_DAYS);
 const oneYearFromNow = () => nowPlusDays(EXPIRY_DAYS);
 const oneHourAgo = () => Date.now() - 1000 * 60 * 60;
 
+const savedLocation = window.location;
+
 describe('DataAccessRequirements', () => {
   const component = (pageMode?: string) => {
     const path = pageMode
       ? `${DATA_ACCESS_REQUIREMENTS_PATH}?pageMode=${pageMode}`
       : DATA_ACCESS_REQUIREMENTS_PATH;
+    delete window.location;
```

Comment:
This is used in order to mock the switch from react-router query parameters to now utilizing window.location.search. More details can be found below.